_py1/00-getting-started.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "getting started"
+next_lesson:  data types
+next_link: 01-data-types
+group: "fundamentals"
+num: 0
+video_link: "https://www.youtube.com/embed/eeoSWdtRdbE"
+slides_link: "https://docs.google.com/presentation/d/14gBHUUnuMlXLoHfrBnRulRxCLceZ0Asz9aAaq8rDu0A/edit?usp=sharing"
+length: 14
+agenda: <ul class="py1-lesson-brief-agenda"> 
+            <li> what is python? </li>
+            <li> why is it important? </li> 
+            <li> how is it used? </li> 
+        </ul>
+ 
+p1: 
+    question: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat."
+    solution: base-sol.html
+
+p2: 
+    question: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat."
+    solution: base-sol.html
+
+p3: 
+    question: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat."
+    solution: base-sol.html
```

Comment:
Similarly, I would consider making this an array of objects; then, you can have a flexible amount of questions per lesson, and you don't need to hardcode things like `p1 or p2 or p3` in the liquid code!